Public/Src/Cache/ContentStore/Library/Utils/BandwidthChecker.cs

```diff
@@ -94,8 +106,8 @@ async Task impl()
                             var position = destinationStream.Position;
 
                             var receivedMiB = (position - previousPosition) / BytesInMb;
-                            var currentSpeed = receivedMiB / _checkInterval.TotalSeconds;
-                            if (currentSpeed < minimumSpeedInMbPerSec)
+                            var currentSpeed = receivedMiB / _config.BandwidthCheckInterval.TotalSeconds;

```

Comment:
I hate copy pasting stuff, but I've make another change in this logic here: [https://dev.azure.com/mseng/Domino/_git/CloudBuild/pullrequest/507021?_a=files](https://dev.azure.com/mseng/Domino/_git/CloudBuild/pullrequest/507021?_a=files)

I think it make sense to adopt it.

And I would suggest to use the structure from the CB with                 a few helpers like: TraceCopyTaskFailures, TryGetPosition and CheckSufficientBandwidth.